Src/BootCamp.Chapter/Logging/LogToText.cs

```diff
@@ -0,0 +1,41 @@
+﻿using System;
+using System.IO;
+
+namespace BootCamp.Chapter.Logging
+{
+    abstract class LogToText : ILog, IConnection
+    {
+
+        private string _Connection = @"C:\Users\Max\Source\Repos\CSharp-From-Zero-To-Hero\Src\BootCamp.Chapter\Logging\Log.txt";
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Make '_Connection' 'readonly'.](https://app.codacy.com/manual/Almantask/CSharp-From-Zero-To-Hero/pullRequest?prid=5150047)